src/LinkDotNet.Blog.Web/Features/AboutMe/AboutMePage.razor

```diff
@@ -1,19 +1,19 @@
 ﻿@page "/AboutMe"
 @using LinkDotNet.Blog.Web.Features.AboutMe.Components
-@inject AppConfiguration AppConfiguration
+@inject IOptions<ApplicationConfiguration> AppConfiguration
 @inject AuthenticationStateProvider AuthenticationStateProvider
 @inject NavigationManager NavigationManager
-@if (AppConfiguration.IsAboutMeEnabled)
+@if (AppConfiguration.Value.IsAboutMeEnabled)
 {
-    <OgData Title="@("About Me - " + AppConfiguration.ProfileInformation.Name)"
-            Description="@("About Me," + AppConfiguration.ProfileInformation.Name)"
-            Keywords="@AppConfiguration.ProfileInformation.Name"
+    <OgData Title="@("About Me - " + AppConfiguration.Value.ProfileInformation.Name)"
```

Comment:
That keeps me thinking if we want to disect this a bit further.
So we have smaller options instead of this one main root object. So the AboutMe page would retrieve a IOptions<ProfileInformation> ... but this can be a follow up